presslabs/wordpress-integration
# Serve upload http

This issue is fixed in https://github.com/presslabs/php-runtime and this is just the integration part.
It removes `nginx-lua` and adds a new `rclone` container in the docker-compose.

Fixes https://github.com/presslabs/wordpress-integration/issues/11

##### Test
- `git checkout serve-upload-http`
- `docker-compose kill && docker-compose rm`
- `docker rmi $(docker-compose images | grep wordpress_1 | awk '{print $4}')`
- `docker-compose up`
- install wordpress
- upload an image
- in WP, you **should** see the newly uploaded image
- delete it from the GCS bucket
- in WP, you **should** receive 404
